src/OnlineSales/Interfaces/CommentConrollerService.cs

```diff
@@ -0,0 +1,91 @@
+﻿// <copyright file="CommentConrollerService.cs" company="WavePoint Co. Ltd.">
+// Licensed under the MIT license. See LICENSE file in the samples root for full license information.
+// </copyright>
+
+using AutoMapper;
+using AutoMapper.Configuration.Conventions;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Nest;
+using OnlineSales.Controllers;
+using OnlineSales.Data;
+using OnlineSales.DTOs;
+using OnlineSales.Entities;
+using OnlineSales.Helpers;
+using OnlineSales.Services;
+
+namespace OnlineSales.Interfaces;
+
+public class CommentConrollerService
```

Comment:
@IgorKostyukov it is not a service. I think it is better to call it something like CommentableControllerExtension and put it into Controllers folder.